contracts/ERC404.sol

```diff
@@ -138,7 +142,7 @@ abstract contract ERC404 is IERC404 {
   ) public virtual returns (bool) {
     // The ERC-721 tokens are 1-indexed, so 0 is not a valid id and indicates that
     // operator is attempting to set the ERC-20 allowance to 0.
-    if (valueOrId_ <= _minted && valueOrId_ > 0) {
+     if (isValidValueOrId(valueOrId_)) {
```

Comment:
are there other places this logic is used or is this the only place we can use this function?

```suggestion
     if (_isTokenId(valueOrId_)) {
```